### ISSUE TITLE
Feature Request - Make Rate Limits Runtime Changeable 

### ISSUE THREAD

**Issue by AlexCWorrall** [lozzd/octopus-energy-rates-card#44]
The rate limits can currently only be set, hardcoded, whilst editing the dashboard. 

Would be great if they could be also have the ability to be set by an entity value. For example, I trigger charging my batteries based on an input number which specifies the rate at which to start or stop charging. If I could use that value as a rate limit (likely lowlimit or midlimit), I would be able to see the rate card coloured based in whether my batteries would charge or not within each time period. As I change the input rate, the rate card would dynamically change, giving me quick visual feedback of how many 30 mins slots I would be charging in.

**Comment by corvus2606** [comment]
My latest pull request #61 includes a change that allows an entity to be specified with multipliers for medium and high limits.
```
limitEntity: sensor.electric_average_unit_rate
mediumLimitMultiplier: 0.8
highLimitMultiplier: 1.1
```

**Comment by lozzd** [comment]
This is merged and live in v0.5.0
